.travis.yml

```diff
@@ -1,14 +1,42 @@
 language: php
+sudo: required
 
 php:
-  - 5.5
   - 5.6
 
+services:
+  - mysql
+
+env:
+  - DB=mysql
+  
+before_install:
+  # Make sure everything's up to date.
+  - sudo apt-get update -qq
+  - travis_retry composer selfupdate
+
+install:
+  # Install Ansible.
+  - sudo pip install ansible
+  - travis_retry composer install --no-interaction
+  - wget http://ua2.php.net/get/php-5.6.16.tar.gz/from/this/mirror -O php-5.6.16.tar.gz
+  - tar -xzf php-5.6.16.tar.gz
+  - cd php-5.6.16/ext/pcntl/
+  - phpize && ./configure && make install
```

Comment:
Should it be added to config file?

```
$ cd modules
$ cp pcntl.so /usr/lib/php5/<long number like 20090626>/
$ sudo gedit /etc/php5/conf.d/pcntl.ini
```
http://j7mbo.blogspot.com/2013/05/install-pcntl-in-php5-on-ubuntu.html